src/Notebooks/Tutorial.dib

```diff
@@ -302,8 +302,6 @@ let headingStyle =
 
 You can set the font name using `FontName` and the size using `FontSize`:
 
-<!-- Test -->
```

Comment:
This depends on the fonts installed on a given env. I propose to not use this as a test as it is too brittle.